src/python/WMCore/REST/Main.py

```diff
@@ -85,29 +85,28 @@ def access(self):
             delta_time = (time.time() - request.start_time)*1e6
         else:
             delta_time = 0
-        self.access_log.log \
-          (logging.INFO,
-           ('%(t)s %(H)s %(h)s "%(r)s" %(s)s'
-            ' [data: %(i)s in %(b)s out %(T).0f us ]'
-            ' [auth: %(AS)s "%(AU)s" "%(AC)s" ]'
-            ' [ref: "%(f)s" "%(a)s" ]') %
-           { 't': self.time(),
-             'H': self.host,
-             'h': remote.name or remote.ip,
-             'r': request.request_line,
-             's': response.status,
-             # request.rfile.rfile.bytes_read is a custom CMS web
-             # cherrypy patch not always available, hence the test
-             'i': (getattr(request.rfile, 'rfile', None)
-                   and getattr(request.rfile.rfile, "bytes_read", None)
-                   and request.rfile.rfile.bytes_read) or "-",
-             'b': nout or "-",
-             'T': delta_time,
-             'AS': inheaders.get("CMS-Auth-Status", "-"),
-             'AU': inheaders.get("CMS-Auth-Cert", inheaders.get("CMS-Auth-Host", "")),
-             'AC': getattr(request.cookie.get("cms-auth", None), "value", ""),
-             'f': inheaders.get("Referer", ""),
-             'a': inheaders.get("User-Agent", "") })
+        msg = ('%(t)s %(H)s %(h)s "%(r)s" %(s)s'
```

Comment:
where does this msg is used now? I don't see any place here. Am I missing something?